iroh-net/src/endpoint.rs

```diff
@@ -502,6 +507,10 @@ impl Endpoint {
     /// information being provided by either the discovery service or using
     /// [`Endpoint::add_node_addr`].  See [`Endpoint::connect`] for the details of how it
     /// uses the discovery service to establish a connection to a remote node.
+    #[deprecated(
+        since = "0.27.0",
+        note = "Please use `connect` directly with a `NodeAddr` or `NodeId`"
```

Comment:
can we add when we'll be removing this?

when will we be removing it?  :grin:

Reply:
Added 0.28.0. If we remove it one later nobody will be super unhappy I guess...